test/utils.ts

```diff
@@ -30,7 +30,11 @@ export async function renderPage (path = '/') {
     pageErrors.push(err)
   })
   page.on('request', (req) => {
-    requests.push(req.url().replace(url('/'), '/'))
+    try {
+      requests.push(req.url().replace(url('/'), '/'))
+    } catch (err) {
+      // TODO
```

Comment:
what kind of error was being triggered here?

Reply:
The lovely context is not available one! Turns out we somehow still keep loading the Browser pages after tests are being done and we don't have context anymore.

This was more visible in new nuxi-edge case because we early close server properly now and made it more visible.

However i think would be nice later investigate root causes. Seems we are wasting too much on browser allocation in our tests.

Reply:
nice find!